api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -750,22 +750,52 @@ message ScopedRds {
       [(validate.rules).message = {required: true}];
 }
 
+// [#next-free-field: 6]
 message HttpFilter {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.network.http_connection_manager.v2.HttpFilter";
 
+  // [#not-implemented-hide:] Configuration source specifier for the late-bound
+  // filter configuration. The HTTP Listener is warmed until all the initial
+  // filter configurations are received, unless the flag to apply the default
+  // configuration is set. Subsequent filter updates are atomic, and apply to
+  // new streams while the active streams continue using the older filter
+  // configurations.
+  // If the initial delivery of the filter configuration fails, due to a
+  // timeout for example, the optional default configuration is applied.
+  // Without a default configuration, the filter is disabled, and the HTTP
+  // listener responds with 500 immediately. After the failure, the listener
+  // continues subscribing to the subsequent filter configurations.
+  message HttpFilterConfigSource {
+    config.core.v3.ConfigSource config_source = 1;
```

Comment:
Can you clarify how resources are named in the FDS? Presumably this is the based on the HTTP filter name, not the resource type or in-built filter name (e.g. `envoy.filter.http.foo`).

Reply:
It uses the name field in the filter config. The name can be arbitrary, since the typed config is used to look up extensions. I can clarify this bit.

Reply:
Updated to use envoy.config.core.v3.TypedExtensionConfig.